src/release/main.d

```diff
@@ -80,15 +80,23 @@ void main ( string[] params )
         .releasedTags()
         .map!(a=>GithubReleaseVersion(Version.parse(a.name)).ifThrown(GithubReleaseVersion(InvalidVersion.init)))
         .filter!(a=>a.peek!Version)
-        .map!(a=>a.get!Version)
+        .map!(a=>a.get!Version);
+
+    auto tags_with_rc = tags
+        .filter!(a=>a.prerelease.length > 0 || a.metadata.length == 0)
+        .array;
+
+    auto tags_no_rc = tags
         .filter!(a=>a.prerelease.length == 0 && a.metadata.length == 0)
         .array;
 
-    sort(tags);
+    sort(tags_with_rc);
+    sort(tags_no_rc);
+
 
     cmd("git remote update");
 
-    auto myrelease = autodetectVersions(tags);
+    auto myrelease = autodetectVersions(tags_with_rc);
```

Comment:
I would refer to pre-releases as `prerelease` instead of `rc` here (and as "pre-release" for user-oriented text), even when at first we plan to use this for RCs, I think at some point we might want to release a beta or even an alpha to maybe test some application or something.